src/config/derives.rs

```diff
@@ -0,0 +1,53 @@
+use super::ident::Ident;
+use super::parsable::Parse;
+use toml::Value;
+use super::error::TomlHelper;
+
+#[derive(Clone, Debug)]
+pub struct Derive {
+    pub ident: Ident,
```

Comment:
Better `pub name: String` instead, but then possible you need add manual parse.

Reply:
`impl<T: Parse + AsRef<Ident>> Parsable for Vec<T> {` just don't need `AsRef` so it IMHO will be applied automatically.